skins/oasis/css/core/breakpoints-variables.scss

```diff
@@ -4,11 +4,13 @@
 $widthType: get_command_line_param( "widthType", 0 );
 $oasisTypography: get_command_line_param( "oasisTypography", 0 );
 
-$breakpoint-width-not-supported: 767px;
+$scroll-bar-width: 20px;
```

Comment:
Can you add a comment where did this value come from? I can see some articles like http://www.textfixer.com/tutorials/browser-scrollbar-width.php